src/dcp/dcp.c

```diff
@@ -202,6 +322,40 @@ int main(int argc, char** argv)
                         mfu_copy_opts->grouplock_id);
                 }
                 break;
+#endif
+#ifdef DAOS_SUPPORT
+            case 'x':
+	        rc = uuid_parse(optarg, src_pool_uuid);
+                if (rank == 0 && rc != 0) {
+                    MFU_LOG(MFU_LOG_ERR, "Failed to parse source pool uuid: '%s'", optarg);
```

Comment:
I think we should be setting usage=1 on errors when processing the options.  Otherwise, I think it will try to execute anyway.  Be sure to set usage=1 on all ranks, though, while only rank 0 prints the error.